### DISCUSSION TITLE
Version 9 doesn't show download percentage complete

### DISCUSSION THREAD

**Discussion by rw330i** [aclap-dev/video-downloadhelper#701] [Help and Q&A]
### Your question

I have a download in progress that has been downloading for 25 minutes. Is it almost done? halfway? What percentage is complete? I can't tell. In version 8 I could then I could decide whether to wait, restart it, or cancel. Thanks.

### Addon & CoApp version and details

version: 9.0.1.22
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"d83fe2df0e624dbd"}
platform: x86-32 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "C:\\Downloads\\Video",
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_wide_ui": false,
    "use_sidebar": false,
    "open_count_store": 10,
    "successfull_dl": 9
}

**Comment by paulrouget** [comment]
There's a progressbar, no?
Or, you see the progressbar, but it's bouncing right and left?

**Reply by rw330i** [reply]
I just found that the progress bar (% complete) is shown if I go back to the old UI. The progress bar is not shown in the new UI.

**Reply by paulrouget** [reply]
I'm sorry, but I don't understand. Can you take a screenshot of the "no progressbar" in the new UI?

**Reply by rw330i** [reply]
New UI - On a short video you see the progress but on a long video the progress bar just goes back & forth on & on with no idea of how much is left
![New UI](https://github.com/aclap-dev/video-downloadhelper/assets/34139458/50ec8225-0e9c-4744-9fb8-ffa3b5177a2c)
Old UI -
![Old UI](https://github.com/aclap-dev/video-downloadhelper/assets/34139458/f81b131f-4b61-4ea3-a766-199e3dfaa82c)

I can't tell if this worked.

**Reply by rw330i** [reply]
Not my style. I just grabbed the first long video I could find.

**Reply by paulrouget** [reply]
> the progress bar just goes back & forth

That what happens when we actually don't know how long it will take. Sometimes the server tells us a size, we download it, and discover there's more to it.

We just can't tell the length, so we go into that "unknown" state.

Previous versions would make a guess, but that guess was incorrect. But I can investigate a bit more see if, indeed, we were getting the right guess in the old version but not in the new version.

Mind sharing the URL of the video? (just pick a random one that exhibits that behavior)

**Comment by jeff15110168** [comment]
my bar is also bouncing...the old version perfectly showed the estimated time and it was indeed always correct. can this be fixed back?

**Reply by paulrouget** [reply]
> my bar is also bouncing...the old version perfectly showed the estimated time and it was indeed always correct. can this be fixed back?

I'll look, but please share the URL where it's bouncing?

**Comment by rw330i** [comment]
The guess may not have been completely accurate but it provided valuable information. I could see the download rate, about what percentage had been completed, and most important, get an idea of how much longer the download would take. If I was downloading a 520MB file and after 20 minutes I could see that only 10% had been downloaded or that the rate was slow or slowing then I knew the download was not worth continuing. It also allowed me to easily see downloads that were hung at a certain percentage.
Whatever you do in the future please don't be like Chrome. Always give us the option to go back to the old UI. That is what I have done now.
